packages/projection-typeorm/src/operators/storeCredentials.ts

```diff
@@ -0,0 +1,29 @@
+import { ChainSyncEventType } from "@cardano-sdk/core";
+import { CredentialEntity, typeormOperator } from "../../dist/esm";
+import { uniq } from "lodash";
+import { Mappers } from "@cardano-sdk/projection";
+import { credentialsFromAddress } from '@cardano-sdk/core/src/Cardano/util';
+
+export const storeCredentials = typeormOperator<Mappers.WithUtxo>(async (evt) => {
+  const {
+    block: { body: txs },
+    eventType,
+    queryRunner
+  } = evt;
+
+  // produced credentials will be automatically deleted via block cascade
+  if (txs.length === 0 || eventType !== ChainSyncEventType.RollForward) return;
+  
+  const involvedOutputCredentials = uniq(evt.utxo.produced.map(([_, txOut]) => txOut.address).map((address) => {
+    const credential = credentialsFromAddress(address);
```

Comment:
That's fine for POC but I consider this tech debt. It should depend on `WithAddresses` mapper instead (`typeormOperator<Mappers.WithUtxo & WithAddresses>`). We just need to add functionality to group addresses by tx id there.